UnityProject/Assets/Scripts/UI/ControlChat.cs

```diff
@@ -73,6 +73,7 @@ public class ControlChat : MonoBehaviour
 	/// Are the channel toggles on show?
 	/// </summary>
 	private bool showChannels = false;
+	private bool debounceCloseChat = false;
```

Comment:
This debounce logic is unnecessary. Behaviour seemed identical with and without it.

Reply:
Are you sure? On my machine the chat never closes when sending a message without it... Maybe its the script laod order?

Reply:
What version of unity are you using, and what OS? Mine closes just fine without it. 
@fomalsd did you experience something like this when you tested before?

Reply:
Windows 10, 2018.3.14f - The README states 2018.3

Reply:
current behaviour on develop is like that:
pressing enter with empty message never closes chat (regardless of focused state), 
pressing esc always closes chat and shows esc menu
pressing enter with non-empty message always sends it (regardless of focused state)